.github/workflows/cmake.yml

```diff
@@ -95,6 +93,29 @@ jobs:
           cmake . -DTEST:STRING="defaults-enabled"
           cmake --build .
 
+  preset-support-but-not-enough:
+    runs-on: ubuntu-22.04
+    steps:
+      - uses: actions/checkout@v4
+      - uses: ./.github/actions/install/cmake
+        with:
+          version: "3.24.4"
+          url: "https://cmake.org/files/v3.24/cmake-3.24.4.tar.gz"
+      - uses: ./.github/actions/install/gtest
+
+      - name: setup
+        run: |
+          mkdir build
+          cd build
+          cmake .. -DJWT_BUILD_EXAMPLES=ON -DJWT_BUILD_TESTS=ON
+          sudo make install
```

Comment:
Is there a reason for the explicit make vs a `cmake --build . --target install` ?

Reply:
most of the are `make install` but there's no reason... it should really be `cmake --install .` for the ones not using presets